internal/expr/parts.go

```diff
@@ -1,69 +0,0 @@
-package expr
-
```

Comment:
Usually we try not to move code and refactor it at the same time, as it makes it hard to review what actually changed here. It ends up as just 70 lines more and 70 lines less on the other side.  It's also unclear how this is improving things.

Reply:
This change is primarily to aid readability rather than cut down on the lines of code. Alberto's comment explains in greater depth.

This is motivated by Albertos impressions on the code coming in as a fresh pair of eyes on it.